compile (link) error "No implementations provided for the following modules"

I'm getting the following error
```
File "_none_", line 1:
Error: No implementations provided for the following modules:
         Cohttp_lwt_body referenced from src/alba.cmx
         Cohttp_lwt_unix referenced from src/alba.cmx
Command exited with code 2.
Compilation unsuccessful after building 399 targets (0 cached) in 00:00:35.
make: *** [build-alba] Error 10
```
while trying to compile the (lwt http client) example from the README

https://github.com/openvstorage/alba/commit/d78586958a316a1176a2671837ea602a1505fb0d
https://api.travis-ci.org/jobs/87009551/log.txt?deansi=true

--- a/README.md
+++ b/README.md
@@ -39,6 +39,17 @@ You can also obtain the development release:
 $ opam pin add cohttp --dev-repo
 ```
 
+### Findlib (Ocamlfind)
+
+Cohttp ships with 6 findlib libraries:
+
+* cohttp - Base `Cohttp` module. No platform specific functionality.
+* cohttp.async - Async backend `Cohttp_async`
+* cohttp.js - Jsoo (XHR) client
+* cohttp.lwt - Unix based lwt backend
+* cohttp.lwt-core - Lwt backend without unix specifics.
+* cohttp.top - Print cohttp types in the toplevel (`#require "cohttp.top"`)
+
 ## Client Tutorial
 
 Cohttp provides clients for Async, Lwt, and jsoo (Lwt based). In this tutorial,

